service/address_utils.py

```diff
@@ -49,10 +57,13 @@ def get_street_name_lines(address_data):
             street_name_string += street_name
     if street_name_string:
         lines.append(street_name_string)
+    LOGGER.debug("get_street_name_lines: {0}".format(lines))
```

Comment:
don't technically need the numbering here. There's one parameter and one placeholder. It's only really useful if you have multiple parameters but then it's still easier to place them in the order you want displayed rather than numbering them.